SOURCEBUILD.md

```diff
@@ -4,27 +4,33 @@ The following instructions are for users wishing to build cuGraph from source co
 
 The cuGraph package include both a C/C++ CUDA portion and a python portion.  Both libraries need to be installed in order for cuGraph to operate correctly.
 
-### Prerequisites
+## Prerequisites
 
-Compiler requirement:
-
-* `gcc`     version 5.4+
-* `nvcc`    version 10.0+
-* `cmake`   version 3.12
-
-CUDA requirement:
+__Compiler__:
+* `gcc`         version 5.4+
+* `nvcc`        version 10.0+
+* `cmake`       version 3.12
 
+__CUDA:__
 * CUDA 10.0+
 * NVIDIA driver 396.44+
 * Pascal architecture or better
 
```

Comment:
No need to add clang-format and flake8 here?

https://github.com/rapidsai/cugraph/pull/859/files#diff-6a3371457528722a734f3c51d9238c13R12

Reply:
Nope - this is just how to compile the code / not modify it.